homeworksyllabus.html

```diff
@@ -0,0 +1,12 @@
+<p>List the rules for collaboration on homework assignments</p>
+<p>You can go to any resources availaible when doing homework. Just make sure to document the source and understand the source so you can write your own answer. But don't do your homework with others outside your group.</p>
+
+<p>List the rules for consulting external references for homework assignment</p>
+<p>when doing homework assignments you can discuss it with your classmates but not anyone outside your group.</p>
```

Comment:
You can discuss with people outside your group but you can't sit down and write code together,